mmv1/products/networkservices/HttpRoute.yaml

```diff
@@ -0,0 +1,507 @@
+                        - rangeMatch
+                    - !ruby/object:Api::Type::Boolean
+                      name: presentMatch
+                      description: |
+                        A header with headerName must exist.
+                      exactly_one_of:
+                        - exactMatch
+                        - regexMatch
+                        - prefixMatch
+                        - presentMatch
+                        - suffixMatch
+                        - rangeMatch
+                    - !ruby/object:Api::Type::String
+                      name: suffixMatch
+                      description: |
+                        The value of the header must end with the contents of suffixMatch.
+                      exactly_one_of:
+                        - exactMatch
+                        - regexMatch
+                        - prefixMatch
+                        - presentMatch
+                        - suffixMatch
+                        - rangeMatch
+                    - !ruby/object:Api::Type::NestedObject
+                      name: rangeMatch
+                      description: |
+                        If specified, the rule will match if the request header value is within the range.
+                      exactly_one_of:
+                        - exactMatch
+                        - regexMatch
+                        - prefixMatch
+                        - presentMatch
+                        - suffixMatch
+                        - rangeMatch
+                      properties:
+                        - !ruby/object:Api::Type::Integer
+                          name: start
+                          description: |
+                             Start of the range (inclusive).
+                          required: true
+                        - !ruby/object:Api::Type::Integer
+                          name: end
+                          description: |
+                             End of the range (exclusive).
+                          required: true                
+        - !ruby/object:Api::Type::NestedObject
+          name: action
+          description: |
+              The detailed rule defining how to route matched traffic.
+          send_empty_value: true
+          properties:
+            - !ruby/object:Api::Type::Array
+              name: destinations
+              description: |
+                The destination to which traffic should be forwarded.
+              item_type: !ruby/object:Api::Type::NestedObject
+                properties:
+                  - !ruby/object:Api::Type::String
+                    name: serviceName
+                    description: |
+                      The URL of a BackendService to route traffic to.
+                  - !ruby/object:Api::Type::Integer
+                    name: weight
+                    description: |
+                      Specifies the proportion of requests forwarded to the backend referenced by the serviceName field.
```

Comment:
Can we copy the whole description from API  here?

https://cloud.google.com/traffic-director/docs/reference/network-services/rest/v1beta1/projects.locations.httpRoutes#Destination